src/util/expr.cpp

```diff
@@ -38,6 +46,11 @@ void exprt::move_to_operands(exprt &e1, exprt &e2)
   op.back().swap(e2);
 }
 
+/// Move the given arguments to the end of `exprt`'s operands.
+/// The argument is destroyed and mutated to a reference to a nil `irept`.
+/// \param e1: first `exprt` to append to the operands
+/// \param e2: second `exprt` to append to the operands
+/// \param e3: second `exprt` to append to the operands
```

Comment:
I'd be inclined to deprecate or outright remove this one. There are currently two users in the code base.

Reply:
Out of the scope of this PR.